src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -492,6 +501,14 @@ public function iChangeItsPriceTo($price, $channelName)
     }
 
     /**
+     * @When /^I change its original price to (?:€|£|\$)([^"]+) for "([^"]+)" channel$/
```

Comment:
We usually put prices inside quotation marks, so `(?:€|£|\$)([^"]+)` -> `"(?:€|£|\$)([^"]+)"`